common4j/src/main/com/microsoft/identity/common/java/controllers/ExceptionAdapter.java

```diff
@@ -236,6 +236,19 @@ public static ServiceException getExceptionFromTokenErrorResponse(@NonNull final
                     developerDescription,
                     apiError
             );
+        } else if (isNativeAuthenticationResetPasswordRequiredError(errorResponse)) {
+            ServiceException apiError = new ServiceException(
+                    errorResponse.getError(),
+                    errorResponse.getErrorDescription(),
+                    null);
+
+            String developerDescription = "Password change is required, which can't be fulfilled as part of this flow."+
```

Comment:
nit: maybe mention the word "user" here, to make it clear this is about a user's password and an action the user has to complete.

Reply:
Thanks. Should this message be "User password change is required, which can't be..."?

Reply:
Sounds good to me.